### PR TITLE
Clean up imports

### PR DESCRIPTION
**Pull request by stramel**
Just some simple clean up of the imports/exports.

- Combines duplicate imports
- Puts all imports before any exports
- Switches `useMutation` to export the default to be consistent with the others in `index.ts`
- Groups package imports before local file imports.

@morrys Feel free to close this if you're not interested in these changes.

### CODE REVIEW

**Comment by morrys** [comment]
it's impossible not to be interested in a clean code :)

@stramel is it okay if I make some changes in your branch? I am updating the versions of eslint and defining new rules to ensure that the code then remains clean :)

**Comment by stramel** [comment]
Yeah go for it!

**Comment by morrys** [comment]
what do you think about it? :D

**Comment by stramel** [comment]
I agree that is a good improvement 👍

File: .eslintrc.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ module.exports = {
         'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
         'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
         'plugin:prettier/recommended', // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+        //'plugin:react-hooks/recommended', TODO add in next release
```
Comment (morrys) [inline .eslintrc.js]: @stramel now it's the turn of this rules :)

**Pull request merged**